eng/sdl-tsa-vars.config

```diff
@@ -0,0 +1,11 @@
+-SourceToolsList @("policheck","credscan")
+-TsaInstanceURL https://devdiv.visualstudio.com/
+-TsaProjectName DEVDIV
+-TsaNotificationEmail aspnetcore-build@microsoft.com
+-TsaCodebaseAdmin REDMOND\kevinpi
+-TsaBugAreaPath DevDiv\ASP.NET Core
```

Comment:
@jcagme does this need to be quoted, since it has a space?

Reply:
Both cases work, but in summary, no quotes needed